src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -523,6 +523,10 @@
   <data name="ToolSettingsUnsupportedRunner" xml:space="preserve">
     <value>Command '{0}' uses unsupported runner '{1}'."</value>
   </data>
+  <data name="SolutionExecutableConfigurationMismatchError" xml:space="preserve">
+    <value>Multiple executable projects in the solution contain conflicting '{0}' values. Make the values match. Consider using a Directory.build.props file to set all project configurations.  Conflicting projects:
```

Comment:
Maybe `Ensure the values match` instead of `Make`. Also a double space at the end before `Conflicting projects`

Reply:
Also added a period at the end!